code/controllers/subsystem/ticker.dm

```diff
@@ -206,22 +206,10 @@ SUBSYSTEM_DEF(ticker)
 				tipped = TRUE
 
 			if(timeLeft <= 0)
-				if(!checkreqroles())
```

Comment:
Okay wow this sucked- but note that code may expect certain roles to be present, and may runtime without them.